packages/lesswrong/components/vulcan-forms/Form.tsx

```diff
@@ -55,7 +55,7 @@ import withCollectionProps from './withCollectionProps';
 // props that should trigger a form reset
 const RESET_PROPS = [
   'collection', 'collectionName', 'typeName', 'document', 'schema', 'currentUser',
-  'fields', 'removeFields',
+  'fields', 'removeFields', 'documentId',
```

Comment:
What is this change?

Reply:
When using the "next tag" button to iterate through tags, each one taking you the next tag with editing_state = true, if you don't have this, the editor contents don't change to the next tag.

Reply:
Oops, sorry, ignore it. Should be removed.

Reply:
```suggestion
  'fields', 'removeFields',
```